geniushubclient/zone.py

```diff
@@ -221,18 +344,257 @@ def footprint_schedule(raw_json) -> Dict:
 
         return self._data
 
+    def update_state(self, json):
```

Comment:
The main reason that I left them was that I wasn't sure if these methods would eventually be able to be called individually by a client, hence needing them to not be private. However, there's no reason that they couldn't be (as they're not really private), and if a use case ever comes up for them needing to be public, it can be changed then.

Reply:
Changed these all to private methods.